src/gui/footer/footer_line.cpp

```diff
@@ -51,6 +51,8 @@ bool FooterLine::Create(footer::Item item_id, size_t index) {
 
     if (result) {
         positionWindows();
+    } else if (item_id == footer::Item::none) {
+        Erase(index);
```

Comment:
This doesn't seem to be a correct fix. The result is set to true on any change, which should call positionWindows. That is, even on change to/from none. This thing might possibly work, but it's a bit random, plus it will call positionWindows each time (inside Erase), suboptimally.

By quickly looking at the code, the proper/elegant fix would probably be to add `FooterItemMappingRec<std::monostate, Item::none>` to `FooterItemMappings`

Reply:
Trying to add FooterItemMappingRec as suggested doesn't work due to compiler error:
`error: static assertion failed: T must occur exactly once in alternatives
[build]  1115 |       static_assert(__detail::__variant::__exactly_once<_Tp, _Types...>,`.
As far as I could see, Erase doesn't call positionWindows and it only is erased if result is false, which happens if the item isn't registered (which is only the case for footer::item::none).

Reply:
I'm now using unregister directly and also moved it above the FooterItemsMapping function, which should be a more proper way to do it. If you have any ideas how to make the FooterItemMappingRec working with std::monostate, please let me know :)